chapters/4-acquisition.tex

```diff
@@ -508,12 +530,180 @@ \subsection{Using electronic survey features}
+Designing systematic data checks and running them routinely throughout data collection
+simplifies field monitoring and improves data quality.
+There are two important types of checks for data quality monitoring: 
+high frequency quality checks and field validation.\sidenote{For a comprehensive discussion
+	of monitoring data quality, see the DIME Wiki: \url{https://dimewiki.worldbank.org/Monitoring\_Data\_Quality}}
+
+\textbf{High frequency data quality checks} should be scripted 
+in advance of the start of data collection,
+so that data checks can start as soon as data starts to come in. 
+A research assistant should run the high-frequency checks (HFCs) on a daily basis 
+for the duration of the survey.\sidenote{To learn more about what types of checks should be included in the HFCs,
+	see the DIME Wiki: \url{https://dimewiki.worldbank.org/High\_Frequency\_Checks}}
+HFCs should include monitor consistency and range of responses to each question, 
+survey programming validation, tests for enumerator-specific effects, 
+and checks for duplicate entries and completeness of online submissions vis-a-vis the field log.
+
+High-frequency checks will only improve data quality
+if the issues they catch are communicated to the data provider
+and corrections are documented and applied to the data.
+This requires close communication with the field team, 
+so that enumerators are made aware of data quality issues promptly,
+and a transparent system for document issues and corrections. 
+There are many ways to communicate results of high-frequency checks to the field team.
+What's most important is to find a way to create actionable information for your team.
+\texttt{ipacheck}, for example, generates a spreadsheet with flagged errors;
+these can be sent directly to the data collection teams.
+Many teams choose other formats to display results,
+such as online dashboards created by custom scripts.
+It is also possible to automate communication of errors to the field team
+by adding scripts to link the HFCs with a messaging platform.
+Any of these solutions are possible:
+what works best for your team will depend on such factors as
+cellular networks in field work areas, whether field supervisors have access to laptops,
+internet speed, and coding skills of the team preparing the HFC workflows.
+
+Careful \textbf{field validation} is essential for high-quality survey data.
+While we cannot control natural measurement error,\sidenote{
+	See \citet{kondylis2015measuring} for an example.}
+which comes from variation in the realization of key outcomes,
+there is often an opportunity to reduce error arising from inaccuracies in the data generation process.
+\textbf{Back-checks}, spotchecks and other validation audits help ensure that 
+data is not falsified, incomplete, or otherwise suspect.\sidenote{For instructions
+	on how to design and implement Back Checks, see the DIME Wiki: \url{https://dimewiki.worldbank.org/Back\_Checks}}
```

Comment:
```suggestion
data is not falsified, incomplete, or otherwise suspect.\sidenote{
  More details on how to design and implement Back Checks
  can be found on the DIME Wiki:
  \url{https://dimewiki.worldbank.org/Back\_Checks}}
```